transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollSocketChannelConfig.java

```diff
@@ -297,6 +303,14 @@ public EpollSocketChannelConfig setTcpKeepCntl(int probes) {
         return this;
     }
 
+    /**
+     * Set the {@code TCP_MD5SIG} option on the socket. See {@code linux/tcp.h} for more details.
+     */
+    public EpollSocketChannelConfig setTcpMd5Sig(Map<InetAddress, byte[]> keys) {
+        channel.setTcpMd5Sig(keys);
```

Comment:
Is it a bad idea to add a getter for this property? It seems to me that it's doable because we keep it as a member field.